bitnami/airflow-worker/2/debian-11/rootfs/opt/bitnami/scripts/libairflowworker.sh

```diff
@@ -46,6 +46,11 @@ airflow_worker_validate() {
 #   None
 #########################
 airflow_worker_initialize() {
+    # Remove airflow-worker.pid file if exists to prevent error after WSL restarts
+    if [ -f $AIRFLOW_PID_FILE ]; then
```

Comment:
To prevent linter errors, could you use double quotes ? 
```suggestion
    if [ -f "$AIRFLOW_PID_FILE" ]; then
```